README.md

```diff
@@ -1,86 +1,99 @@
-All help is appreciated, but there's little point trying to 'play' is as yet
-
-[See the BIG TODO LIST for more info](TODO.md)
-
-# Current build status:
-Linux: [![Linux Build Status](https://travis-ci.org/JonnyH/OpenApoc.svg?branch=master)](https://travis-ci.org/JonnyH/OpenApoc)
-Windows: [![Windows Build Status](https://ci.appveyor.com/api/projects/status/07ndsvrbyct924a1)](https://ci.appveyor.com/project/JonnyH/openapoc)
- 
-# Contact us
-If you're interested, please visit us at http://openapoc.org
-We have forums at http://openapoc.pmprog.co.uk - please pop by and introduce yourself!
-We also have an IRC channel on http://freenode.net - #openapoc
- 
+All help is appreciated, but there's little point trying to 'play' it as of yet.
+
+[See our Trello TO-DO list for more info.](https://trello.com/b/lX5Y3DwR/openapoc)
+
+## Requirements
+
+* This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be acquired from [Steam](http://store.steampowered.com/app/7660/) - this is _required_ to run)
+
 ## Building
+
 OpenApocalypse is built leveraging a number of libraries - to provide needed functionality (and save us the time of implementing it ourselves badly)
- 
-- [SDL2](http://www.libsdl.org)
-- [Boost] (http://boost.org) - specifially the 'locale' library, used for localisation, 'program-options' for settings management, and 'filesystem'
-- [Libunwind] (http://www.nongnu.org/libunwind/download.html) - debug backtracing on linux - not needed on windows
+
+* [SDL2](http://www.libsdl.org)
+* [Boost](http://boost.org) - specifially the 'locale' library, used for localisation, 'program-options' for settings management, and 'filesystem'.
+* [Libunwind](http://www.nongnu.org/libunwind/download.html) - debug backtracing on linux - not needed on windows.
 
 The following libraries are also used, but are shipped as submodules in the repository and directly included in the build, so you don't need to install these dependencies to build or use openapoc.
 
-- [GLM](http://glm.g-truc.net) - Maths library
-- [libsmacker](http://libsmacker.sourceforge.net/) - Decoder for .smk video files
-- [lodepng](http://lodev.org/lodepng/) - Reading/writing PNG image files
-- [miniz](https://github.com/richgel999/miniz) - Zlib-comptible compression library
-- [physfs](https://icculus.org/physfs/) - Library for reading data from .iso files or directory trees (Note: We use a patched version, available at https://github.com/JonnyH/physfs-hg-import/tree/fix-iso - that is required to read the .iso files we use)
-- [pugixml](http://http://pugixml.org/) - XML library used for reading/writing the game data files
-- [tinyformat](https://github.com/c42f/tinyformat) - A c++ typesafe string formatting library.
-
-### Requirements:
-- This assumes that you have the file 'cd.iso' - a copy of the original X-Com Apocalypse CD (This can be got from steam for a pittance http://store.steampowered.com/app/7660/ - this is _required_ to run)
```

Comment:
Perhaps add a section saying "If you have the *real* cd, you can copy all the included files to a directory called "cd.iso" and that would work too"